image_transport/include/image_transport/simple_subscriber_plugin.hpp

```diff
@@ -117,6 +117,16 @@ class SimpleSubscriberPlugin : public SubscriberPlugin
     const std::string & base_topic,
     const Callback & callback,
     rmw_qos_profile_t custom_qos)
+  {
+    this->subscribeImpl(node, base_topic, callback, custom_qos, rclcpp::SubscriptionOptions());
+  }
+
+  virtual void subscribeImpl(
+    rclcpp::Node * node,
+    const std::string & base_topic,
+    const Callback & callback,
+    rmw_qos_profile_t custom_qos,
+    rclcpp::SubscriptionOptions options)
```

Comment:
nit: prefer using `override` and not `virtual` here:

```suggestion
  void subscribeImpl(
    rclcpp::Node * node,
    const std::string & base_topic,
    const Callback & callback,
    rmw_qos_profile_t custom_qos,
    rclcpp::SubscriptionOptions options) override
```